tests/Run-Tests.ps1

```diff
@@ -186,7 +190,7 @@ function ExecTest($argList)
 if ($TestDirectory -eq "")
 {
     $MyPath = (get-item $MyInvocation.MyCommand.Path).Directory.FullName;
-    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Tests"
+    $TestSrcDirectory = Join-Path $MyPath "..\build\$Platform\$Config\Universal Windows\"
```

Comment:
We'll need to update our lab tests to account for this.  
Should be a 1 line fix (famous last words) in Atlas studio.
But, we'll want to checkin the change in a winobjc_internal packaging branch.


---
In reply to: [102313367](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102313367) [](ancestors = 102313367,102312399)